core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -108,6 +110,10 @@ private[spark] class ExecutorAllocationManager(
   private val cachedExecutorIdleTimeoutS = conf.getTimeAsSeconds(
     "spark.dynamicAllocation.cachedExecutorIdleTimeout", s"${Integer.MAX_VALUE}s")
 
+  // whether or not to try and save cached data when executors are deallocated
+  private val replicateCachedData =
+    conf.getBoolean("spark.dynamicAllocation.recoverCachedData", defaultValue = false)
```

Comment:
Please add a config constant in `core/src/main/scala/org/apache/spark/internal/config/package.scala`.

Reply:
added